scripts/utils.mjs

```diff
@@ -2808,6 +2808,7 @@ export function endGroup() {
   } else {
     console.groupEnd();
   }
+  console.log();
```

Comment:
Necessary?

Reply:
yes because when ASAN is tripped it doesnt print a trailing newline before exiting the program